config/example.yaml

```diff
@@ -5,36 +5,157 @@ warnings: ignore
 cores: 4
 quiet: false
 file_ext: .spm
-channel: Height
-amplify_level: 1.0
 filter:
-  threshold:
-    method: std_dev # Options : otsu, std_dev, absolute
-    otsu_multiplier: 1.0
-    std_dev: 1.0
-    absolute:
-      - 1
-      - -1
+  run: true # Options : true, false
+  channel: Height
```

Comment:
I moved it here because currently `channel` is used within `Filters` to extract the data.

I think you're right that it should be moved and that could be part of the work done when we start supporting more image formats, as you say a `loading` section that also configures those options.

Best left for future work as that is an existing issue #137 .